yspryn/hw6/min_max.py

```diff
@@ -0,0 +1,46 @@
+from operator import lt, gt
+
+
+def min_ysp(*args, key=None):
+    key = key or (lambda i: i)
+    res = args[0]
+    for i in args:
+        if key(i) < key(res):
+            res = i
+    return res
+
+
+def max_ysp(*args, key=None):
+    key = key or (lambda i: i)
+    res = args[0]
+    for i in args:
+        if key(i) > key(res):
+            res = i
+    return res
+
+
+def sorted_bubbles_method(*args, key=None, reverse=False):
+    key = key or (lambda i: i)
+    compare = lt if reverse else gt
+    sorted_list = list(args)
+    stop_mark = True
+    while stop_mark:
```

Comment:
i think it does not need stop mark and while loop, you need  to iterate through each index,